jobs/elasticsearch_alerts/templates/cluster.alerts

```diff
@@ -0,0 +1,37 @@
+ALERT ElasticUp
+  IF elasticsearch_up != 1
+  FOR <%= p('elasticsearch_alerts.cluster_down.evaluation_time') %>
+  LABELS {
+    severity = "alert",
```

Comment:
To maintain consistency with all other alerts, can this be `warning` or `critical`? `alert` does not provide much info, as it is already an alert :)